### DISCUSSION TITLE
Table disappeared

### DISCUSSION THREAD

**Discussion by PauloAlves311** [AntlerVC/firetable#1341] [Support / Q&A] [answered]
I left the browser open for a bit. When I came back, my table dissapeared. I had custom functions and more. But now it is asking me to import data??

**Comment by PauloAlves311** [comment]
I can still see the table in the Tables tab. And can see the description that I did. But when I enter it. No data there.

**Comment by BeeBombshell** [comment, marked as answer]
Hello @PauloAlves311, would you mind attempting to sign out, clearing your browser cache, and then logging back in?
